app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+class CommentsController < ApplicationController
+  #index
+  def index
+  @post = Post.find(params[:post_id])
+  @comment = Comment.all
```

Comment:
This will give you all the comments. Seems like you only want all the comments that belong to a particular post though.

```rb
@comment = @post.comments.all
```